Bottom: 20fe7c6973d7285ef1da4d96fb7223c597d0896d
Top:    3c3a98d4ee95978cca9727aa82b5ac077cccaa81
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-22 16:40:46 +0000

buffer.c: Don't re-mask FOUNDFLAG of a copied element

The FOUNDFLAG is already masked by CopyElementLowlevel(), so don't
bother repeating it here.


---

diff --git a/src/buffer.c b/src/buffer.c
index d311420..69f9beb 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -225,12 +225,12 @@ AddElementToBuffer (ElementType *Element)
       END_LOOP;
       PIN_LOOP (element);
       {
-	CLEAR_FLAG (FOUNDFLAG | ExtraFlag, pin);
+	CLEAR_FLAG (ExtraFlag, pin);
       }
       END_LOOP;
       PAD_LOOP (element);
       {
-	CLEAR_FLAG (FOUNDFLAG | ExtraFlag, pad);
+	CLEAR_FLAG (ExtraFlag, pad);
       }
       END_LOOP;
     }
